tests/src/common/oracle_wrapper.rs

```diff
@@ -106,10 +106,12 @@ pub fn add_feeder<Lpn>(test_case: &mut TestCase<Lpn>, addr: impl Into<String>)
 where
     Lpn: Currency,
 {
+    let oracle = test_case.oracle().clone();
+
     let response: AppResponse = test_case
         .app
         .wasm_sudo(
-            test_case.oracle.clone().unwrap(),
+            oracle,
```

Comment:
[all occurences] please check if it's not more appropriate to take the oracle from the test_case by value